src/amitaRJun20/ReplaceWordFromArrayList.java

```diff
@@ -0,0 +1,47 @@
+/*Program 2 :
+Display on which index Techno is available, Replace all Techno with Credits from given ArrayList, and count total number of Credits.
+["Maulik","Techno" ,"Krishna", "Techno", "Credits", "Techno"]
+
+Output : 
+Techno word index in given arrayList is 1,3,5 
+["Maulik","Credits" ,"Krishna", "Credits", "Credits", "Credits"]
+Total Credits words :- 4*/
+
+package amitaRJun20;
+
+import java.util.ArrayList;
+
+public class ReplaceWordFromArrayList {
+
+	void displayIndexAndReplace(ArrayList<String> listOfElements, String wordToReplace, String replaceWordWith) {
+		System.out.println("The Given ArrayList is : " + listOfElements);
+		System.out.print("\nIndex of word " + wordToReplace + " in Given Arraylist is : ");
+		int count = 0;
+		while (listOfElements.contains(wordToReplace)) {
+			int indexOfWord = listOfElements.indexOf(wordToReplace);
+			System.out.print(indexOfWord + " , ");
```

Comment:
1. minor - System.out.print(indexOfWord + " , "); - Its printing 1 extra ',' in end . Can add if condition to control this